terraform/deployments/govuk-publishing-platform/main.tf

```diff
@@ -60,3 +60,14 @@ locals {
   routerdb_security_group_id   = data.terraform_remote_state.infra_security_groups.outputs.sg_router-backend_id
   log_group                    = terraform.workspace == "default" ? "govuk" : "govuk-${terraform.workspace}"
 }
+
+locals {
+  additional_tags = {
+    chargeable_entity    = "govuk-publishing-platform-${terraform.workspace}"
+    environment          = var.govuk_environment
+    project              = "replatforming"
```

Comment:
We probably won't want this to be "replatforming" ultimately, as Replatforming (as a team and a project) should cease to exist shortly after this solution is in production. It's probably fine for now though.

Reply:
we shall amend it when the project has ended